third_party/rust/bech32/v0_9/README.chromium

```diff
@@ -5,5 +5,5 @@ Version: 0.9.1
 Security Critical: yes
 Shipped: yes
 License: MIT
-License File:
+License File: //brave/third_party/rust/chromium_crates_io/vendor/bech32-0.9.1/../../../bech32/v0_9/LICENSE
```

Comment:
Do we need the `License File` field if the `LICENSE` file is available next to the `README.chromium` file?

Reply:
No, doesn't appear it is needed in this case. I'll update.